src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/health/PrisonApiHealthCheck.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.health
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import java.time.Duration
+
+@Component
+class PrisonApiHealthCheck @Autowired constructor(
+  @Qualifier("prisonApiNoiAuthToken") webClient: WebClient,
+  @Value("\${api.health-timeout-ms}") timeout: Duration
```

Comment:
you've not got a default here and also removed the default from `src/main/resources/application.yml`.  Think you'll need one or the other, probably best to default here